octoprint_mrbeam/static/js/loginscreen_viewmodel.js

```diff
@@ -2,10 +2,23 @@
  * Created by andy on 03/03/2017.
  */
 $(function() {
+    $('#settings-usersDialogAddUserName').attr('data-bind','value: $root.users.editorUsername, valueUpdate: \'afterkeydown\'');
+
     function LoginScreenViewModel(parameters) {
         var self = this;
 
+        window.ko.extenders.lowercase = function(target, option) {
+            target.subscribe(function(newValue) {
+                if(newValue !== undefined) {
+                    target(newValue.toLowerCase());
+                    console.log(newValue)
```

Comment:
please remove or make the log output more explanatory

Reply:
Deleted